### DISCUSSION TITLE
On:issue_comment event to trigger pull_request event

### DISCUSSION THREAD

**Discussion by platisd** [community/community#26696] [Actions]
<p>I am trying to trigger a workflow that will run “as a pull_request event” via a comment in a pull request.<br>
I want to do this so that the status check can be associated with the pull request and if successful to contribute to the verification of the pull request.<br>
The use case is similar to what’s described <a href="https://github.community/t/on-issue-comment-events-are-not-triggering-workflows/16784">here</a>.</p>
<p>To give you an idea of what I am trying to achieve exactly, I want to post a checklist to a pull request once a PR is opened. Then the maintainers to manually check all the boxes in the posted comment by the workflow and eventually for the workflow to go green.</p>
<p>Since (very sadly) the issue_comment workflows are not taken into consideration in the same way as the pull request ones, I am trying to trigger a pull request event from an issue_comment one, by changing the body/description of the pull request.<br>
<strong>Unfortunately, this does <em>not</em> trigger an “edited” pull request event as one would expect</strong>.<br>
When I manually, via the browser, go and make the very same change that I am programmatically doing, to the pull request body, then the “edited” pull request event gets triggered as expected and everything works.</p>
<p>TL;DR: When I have my bot make a change to a pull request, this does not trigger an “edited” pull request event.</p>
<p>Is this a bug and if so, is there a workaround?</p>

**Comment by jsoref** [comment, marked as answer]
<p>Odds are you’re using the <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow"><code>GITHUB_TOKEN</code> which doesn’t trigger cascading workflows</a>.</p>
<p>You’ll need a PAT:</p>
<p><a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token" class="onebox" target="_blank" rel="noopener">https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token</a></p>

**Comment by platisd** [comment]
<p>Thanks <a class="mention" href="/jsoref">@jsoref</a> I wasn’t aware of this limitation and using a PAT did the trick.</p>
